### PR TITLE
fix sonar labels format 

### PR DESCRIPTION
**Pull request by vim-zz**
so it's unique text per issue type and rating to better match github labels usage.

the issue it address is:
1. when the issue count is in the label, it creates too many labels that makes it harder to search for in GitHub and LinearB.
2. When the label color changes for same label text (in different PRs) the color will change to all occurrences of this label, as GitHub keeps the label text as a global label. Every time the color changes it leads to undesired chnage of colors of existing labels in old PRs.

The solution is to have the rating in the label text so the color is unique, and to omit the count from the label. For example:
`sonar:vulnerabilities-E` wil be always red, while `sonar:vulnerabilities-C` will be always orange.

### CODE REVIEW

**Review by PavelLinearB — APPROVED** [review]

File: docs/downloads/automation-library/integrations/sonar/label_sonar.cm [hunk 1]
```diff
@@ -11,7 +11,7 @@ automations:
     run:
       - action: add-label@v1
         args:
-          label: '{{ item.icon }} x {{ item.count }} {{ item.name }}'
+          label: '{{ item.icon }} sonar:{{ item.name }}-{{ item.rating }}'
```
Comment (PavelLinearB) [inline docs/downloads/automation-library/integrations/sonar/label_sonar.cm:14]: Should we update the screenshots too in this case?

Comment (vim-zz) [reply docs/downloads/automation-library/integrations/sonar/label_sonar.cm:14]: yes... we should

Comment (BenLloydPearson) [reply docs/downloads/automation-library/integrations/sonar/label_sonar.cm:14]: I'll create one

Comment (BenLloydPearson) [reply docs/downloads/automation-library/integrations/sonar/label_sonar.cm:14]: Updated

**Review by BenLloydPearson — APPROVED** [review]

**Pull request merged**